### ISSUE TITLE
Add GraphQL Template support

### ISSUE THREAD

**Issue by xntrik** [doyensec/PESD-Exporter-Extension#6]
I've raised a PR with some preliminary efforts over https://github.com/doyensec/PESD-Exporter-Extension/pull/5

**Comment by anaximand3r** [comment]
This is very interesting! Currently working on v1.3 with new styling features. I will put this in the schedule for further research/development. Do you have notes/examples of potential enrichments? 
I see the `operationName` being added as note in the preliminary efforts

**Comment by xntrik** [comment]
Yeah, at the time I was interested in easily answering the question: does this appear to be a GraphQL Request (and subsequent response?) And, can we pull the GraphQL Operation Name out of the request. It would be nice if the Operation Name was added as a click/hover link (similar to CSP, JSONParam etc), but I couldn't figure out how to use the PESD API / Interface to do that.

**Comment by xntrik** [comment]
An example of what this looks like
![image](https://user-images.githubusercontent.com/678260/233892948-1b43dda1-d2e0-432c-bdc0-71c5dbc3f50e.png)

**Comment by anaximand3r** [comment]
Now I see. As you have experimented, the current `PESDWrapper` version does not support expandable flags registration by templates. It is something that was in the schedule for later releases, but I will prioritize it :) 
It will be possible to add flags like you did for `GQL_Req`/`GQL_Res` and optionally  pass a data structure that will be placed in the clickable popover 👍
